### PR TITLE
Version 2.0.0

### PR DESCRIPTION
**Pull request by LewisJEllis**
Accepting feedback on the release notes. Will publish first thing tomorrow.

/cc @MaxBittker @benvinegar

### CODE REVIEW

**Comment by benvinegar** [comment]
Based on our previous discussion, this seems right, but I think I'll leave to @MaxBittker to approve.

File: History.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
+# 2.0.0 - 5/10/2017
+- Properly surface errors to preserve process exit conditions [See #308, #257]
+  - Node processes with raven will now exit in *all* the same situations as if raven were not present
```
Comment (MaxBittker) [inline History.md]: maybe don't emphasize all here when it's not a hard hard guarantee

Comment (MaxBittker) [inline History.md]: just say "the same situations" imo

**Comment by LewisJEllis** [comment]
> which points someone needs to be aware of for upgrading?
Mostly just the one point about shutdown conditions; added a bold callout.

Added the postgres autobreadcrumb removal to this.

**Pull request merged**